bootstrap.sh

```diff
@@ -494,8 +494,7 @@ install_volta() {
   if ! command -v volta &>/dev/null; then
     log "Install volta"
     curl https://get.volta.sh | bash
-    # shellcheck disable=SC1090
-    source "$(get_shell_startup_script)"
+    export PATH="${HOME}/.volta/bin:$PATH"
```

Comment:
I think volta also likes its `VOLTA_HOME` environment variable, not sure though -- double check

Reply:
You are right: 

> To manually configure your shell to use Volta, edit your console startup scripts to:
>
> - Set the VOLTA_HOME variable to $HOME/.volta
> - Add $VOLTA_HOME/bin to the beginning of your PATH variable

https://docs.volta.sh/guide/getting-started